### ISSUE TITLE
[public-1.12.3] UI Localization issues list

### ISSUE THREAD

**Issue by Holycrapjimhox** [nmrih/source-game#1312]
NOTE: I'll be using the Spanish translations as I've been constantly keeping them up to date. Also note that some of these issues are very minor and may not be worth fixing, this is intended to just list them to see if they'll be addressed in future updates.

1) Objectives and game_text entities are unlocalized. The current workaround is the map translator SM plugin: https://github.com/dysphie/sm-map-translator
![8](https://user-images.githubusercontent.com/35630284/177440000-61ed1af0-189b-4488-b52a-0e95a573f050.png)

2) The mouse acceleration slider still reads "Low" and "High". All other sliders seem to be translated correctly:
![1](https://user-images.githubusercontent.com/35630284/177437582-6f31efe9-4c93-40c5-a8a3-e9c98e06e714.png)
 

3) Native resolution still reads "(native)" in other languages:
![2](https://user-images.githubusercontent.com/35630284/177438169-68ed1efc-c7ca-47c9-9a8f-53353f233911.png)

4) The "Loading" logo is unlocalized:
![7](https://user-images.githubusercontent.com/35630284/177439025-20e12a3c-f378-4557-9e90-34204ad27c22.png)

5) The "Previous" and "Next" buttons in the character select screen are unlocalized:
![3](https://user-images.githubusercontent.com/35630284/177438246-0cedacc9-51d4-483f-a487-ffdfe752835e.png)

6) The "JOIN GAME" button on the join up screen when entering a server is unlocalized:
![4](https://user-images.githubusercontent.com/35630284/177438283-2731fa0f-7953-4294-ad9d-2c3e25ce7bfd.png)
 

7) The default control scheme image in the loading screen has baked-in English text (and could also need an update to add a few more useful keys):
![5](https://user-images.githubusercontent.com/35630284/177438431-1ceecd5e-ae59-44cf-a654-c2f87aca9556.png)

8) The "Respawns" and "Score" column titles on the player stats are truncated:
![6](https://user-images.githubusercontent.com/35630284/177438753-6a148166-9efc-45e3-b753-712f970d91ab.png)

9) The "WEAPONS", "GEAR", "AMMO", "TAKE" and "CLOSE" lines in the supply drop interface are unlocalized:
![translation](https://user-images.githubusercontent.com/35630284/179549234-97eb11ae-62cc-41f0-9318-0ca7d797a9e5.png)

**Comment by felis-catus** [comment]
As of dev-r9652:
1, 5, 7, & 8 are resolved.
6, & 9 are pending.
Unable to resolve 2, 3, & 4 due to technical restrictions.

**Comment by Blueberryy** [comment]
> As of dev-r9652: 1, 5, 7, & 8 are resolved. 6, & 9 are pending. Unable to resolve 2, 3, & 4 due to technical restrictions.

Yeah as for that "(native)" it might require access to Source Engine because it's hardcoded in.
And as for textures or that loading icon - it could be translated, but will need to create like ``nmrih_textures_*language name*_.vpk`` and someone will have to create translated texture, and more of that, Felis don't have full access to NMRiH files so NMRiH 2 developer will have to help with that.

**Comment by Blueberryy** [comment]
Gotta port screenshots from my issue over here
`"CPU Player"` for bots on Players Muting menu.
![image](https://github.com/user-attachments/assets/8b54473e-e6c1-429a-a316-60542e639098)

`"BOT"` label on scoreboard for bot-players. Recently i saw these 2 texts (`CPU Player` and `BOT`) got localized on [Dystopia ](https://store.steampowered.com/app/17580/Dystopia) source engine game localization update.
![2024_07_21_0n6_Kleki](https://github.com/user-attachments/assets/470b462b-aa2d-4cce-80c1-ba648376eb74)

Default Character names & Voice names.
![image](https://github.com/user-attachments/assets/b23ae5a7-c38f-4237-aa26-864dc05adb9a)

And here's new stuff that i've noticed without localization;
New default server name placeholder `<player name>'s Server`.
<img width="343" height="249" alt="Image" src="https://github.com/user-attachments/assets/de5e8613-3107-4da6-a39e-3567205933ba" />
Version title `v` on main menu.
<img width="163" height="74" alt="Image" src="https://github.com/user-attachments/assets/df806769-ac15-4aa9-a376-9529de761ddb" />

Bink Video copyright breaking into a second line but half of second line text is cut.
<img width="500" height="213" alt="Image" src="https://github.com/user-attachments/assets/164c3a86-1a8a-4079-a4da-4399908e5d77" />

Gamma/game brightness config text
<img width="516" height="402" alt="Image" src="https://github.com/user-attachments/assets/14e4e7dd-f421-4a9a-8c9d-8635d9357211" />

File explorer file sizes (`KB`, `MB`) and change date month texts
<img width="668" height="594" alt="Image" src="https://github.com/user-attachments/assets/58bfee34-c998-4ecb-8822-e1fc00fa42b7" />
